src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -124,28 +125,22 @@ async Task IKernelCommandHandler<SendValue>.HandleAsync(SendValue command, Kerne
 
     private Task SetValueAsync(string valueName, object value, Type? declaredType = null)
```

Comment:
After this PR this function will only be called for VS-specific variables declared in .env files / things like $projectUrl. Is that correct?

Reply:
Yes